slist.sh

```diff
@@ -256,9 +256,10 @@ do
         config-file)
           configfile=true
           val="${!OPTIND}"; OPTIND=$(( OPTIND + 1 ))
+          check_arg "$val"
           # Exit program if host is empty
           if [[ $value == "false" ]]; then
-            printf "%s\n" "${red}Host cannot be empty ${end}"
+            printf "%s\n" "${red}Must specific config file if --config-file is used${end}"
```

Comment:
"Must specify config file if --config-file is used"

Reply:
replace with "Config cannot be empty with --config-file"